### ISSUE TITLE
Train VGG16

### ISSUE THREAD

**Issue by omobayode1** [tensorpack/tensorpack#1490]
I have been trying to train the VGG Model using the vgg.py code.
However, the  Top5 Validation accuracy I am getting is very far from the reported. I am currently using the following settings:
--batch 128
--gpu 4
--norm bn

Please can you provide the settings used during training for the values reported in the repository?

Is it possible to get a model of the VGG model trained with --norm = bn in the model zoo?

Also,
what does --batch parameter?

Is it the number of batch per gpu or the total batch size? The github said something about using batch size of 32 per gpu for the training. This is equivalent to 32*8 for all the GPUs. does it mean batch size is 32 or 256?

Thank you.

**Comment by ppwwyyxx** [comment]
Settings are given in https://github.com/tensorpack/tensorpack/tree/master/examples/ImageNetModels#vgg16 already.
We don't plan to release the models.

> what does --batch parameter?


The above link also says: `see ./vgg16.py --help for usage.`
